test_requirements.txt

```diff
@@ -1,4 +1,6 @@
 pytest >= 6.0.1
 pytest-mock >= 3.2.0
 pytest-cases >= 2.1.2
-coverage >= 5.2.1
\ No newline at end of file
+coverage >= 5.2.1
+aiohttp >= 3.3.2
+aiohttp-cors
```

Comment:
Are these two additions needed?

Reply:
Yes - i was not able to run all the tests witout them :) Some tests rely on AIO stuff, so when i installed them i added them as well :)

Reply:
Aha, [`CONTRIBUTING.md`](https://github.com/psf/black/blob/master/CONTRIBUTING.md) suggests using pipenv, which includes them.

I'm not so familiar with pipenv, but I see Travis CI runs `pip install -e '.[d]'`, where the `d` is a so-called "extra" listed in `setup.py`:
```python
    extras_require={
        "d": ["aiohttp>=3.3.2", "aiohttp-cors"],
        "colorama": ["colorama>=0.4.3"],
    },
```
So that will install them for you. And GitHub Actions uses tox which also installs with the extra.

Want to try running `pip install -e '.[d]'` instead?

Reply:
AFAIK this requirements file is for CI not developers of Black. Doing a Pipenv developement install should install Black with its HTTP web server dependencies. https://github.com/psf/black/blob/407052724fa1c97ee8bcd4e96de650def00be03e/Pipfile#L21 

You get pinned dependencies too so doing a Pipenv development install is recommended

Reply:
oic. I use some...thing.. not actually sure. I simply realized it wasn't exhaustive, not that it was defined elsewhere. Should i remove these two lines?

Reply:
Another popular thing to do is to move tests dependencies from `test_requirements.txt` into a `tests` extra, for example:

* https://github.com/hynek/structlog/blob/69d9e9328d86cdc5e395eadd119c9775ee7343f7/setup.py#L41-L49

But that's for another PR.

Reply:
Yes, I think remove those two lines to avoid duplication and keep things lean 👍

Reply:
Fixed :)